Template.php

```diff
@@ -1496,6 +1496,17 @@ public function expand_by_RIS(&$dat) { // Pass by pointer to wipe this data when
     // Convert &#x__; to characters
     $ris = explode("\n", html_entity_decode($dat, NULL, 'UTF-8'));
     $ris_authors = 0;
+    
```

Comment:
Do we need a `foreach` here?  Could we not instead do a regular expression search along the lines of `preg_match('~\n\s*T[I1]\s+(.*)$', $match)`?